src/layer/AbstractSentinelHubV1OrV2Layer.ts

```diff
@@ -210,4 +213,26 @@ export class AbstractSentinelHubV1OrV2Layer extends AbstractLayer {
     }
     return data;
   }
+
+  public async updateLayerFromServiceIfNeeded(): Promise<void> {
+    if (this.instanceId === null || this.layerId === null) {
+      throw new Error(
+        "One or more of these parameters (polarization, acquisitionMode, resolution) \
```

Comment:
Not every dataset on EOCloud has `polarization, acquisitionMode, resolution` parameters, so I think this message should be changed (maybe to `Some or more dataset-specific parameters are not set ...`).

Reply:
Very true, thanks for noticing! I changed it to `Additional data can't be fetched from service because instanceId and layerId are not defined`. Is it fine with you?

The other comment "assumed" the update was requested because those parameters were not set and thus the layer needed to fetch that additional data. In our case, this is not the premise - in my opion.

Reply:
Great, sounds better than my suggestion. Resolving.